datalad/cmd.py

```diff
@@ -1184,6 +1184,72 @@ def _get_adjusted_env(self, env=None, cwd=None, copy=True):
             copy=False,
         )
 
+    def run_on_filelist_chunks(self, cmd, files, protocol=None,
+                               cwd=None, env=None, **kwargs):
```

Comment:
Should we switch `_call_git` over to this method as well?  Then the only remaining use of `run_gitcommand_on_file_list_chunks` would be in the now unused `_git_custom_command`.

Reply:
YES! Thx, done!